app/plugin_api.go

```diff
@@ -285,6 +285,10 @@ func (api *PluginAPI) SearchChannels(teamId string, term string) (*model.Channel
 	return api.app.SearchChannels(teamId, term)
 }
 
+func (api *PluginAPI) SearchUsers(search *model.UserSearch) ([]*model.User, *model.AppError) {
+	return api.app.SearchUsers(search)
```

Comment:
This method takes more arguments.

Reply:
@csduarte would you please tell what I am missing?

Reply:
The build output can guide you:
```
app/plugin_api.go:289:28: not enough arguments in call to api.app.SearchUsers
	have (*model.UserSearch)
	want (*model.UserSearch, *model.UserSearchOptions)
```

(also I am @crspeller not `csduarte`)